src/Equinor.ProCoSys.IPO.Domain/Events/PreSave/IpoCompletedEvent.cs

```diff
@@ -1,16 +1,20 @@
 ﻿using System;
 using Equinor.ProCoSys.Common;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
 
 namespace Equinor.ProCoSys.IPO.Domain.Events.PreSave
 {
     public class IpoCompletedEvent : IDomainEvent
     {
-        public IpoCompletedEvent(string plant, Guid sourceGuid)
+        public IpoCompletedEvent(string plant, Guid sourceGuid, Participant participant = null)
```

Comment:
Is there a scenario where participant ends up being null? Will we then get a NPE in EventTypeExtensions when having participant.firstName?

Reply:
Originally necessary, but not in the final solution. Removed.